features/cli.feature

```diff
@@ -3,6 +3,21 @@ Feature: Command line interface
   As a person who wants to run features
   I want to run Cucumber on the command line
 
+  Scenario: run with no features directory
+    When I run cucumber.js with `features`
+    Then the error output contains the text:
+    """
+    No such file or directory - features. You can use `cucumber --init` to get started.
+    """
+
+  Scenario: run a nonexistant feature file
+    Given a directory named "features" 
+    When I run cucumber.js with `features/a.feature`
+    Then the error output contains the text:
+    """
+    No such file or directory - a.feature
```

Comment:
Not sure what you mean but I'll take a stab. 

In the first scenario we haven't created a features dir (or a feature file), so it complains and offers the `--init` switch to get you started. 

In the second scenario we have a features dir, but the feature file we are trying to run doesn't exist so we give an error message rather than a stack trace. 

Given those statements can you help me understand what you mean here?